editor/plugins/material_editor_plugin.cpp

```diff
@@ -250,13 +275,13 @@ MaterialEditor::MaterialEditor() {
 	Ref<World3D> world_3d;
 	world_3d.instantiate();
 	viewport->set_world_3d(world_3d); // Use own world.
-	vc->add_child(viewport);
 	viewport->set_disable_input(true);
-	viewport->set_transparent_background(true);
+	viewport->set_transparent_background(false);
 	viewport->set_msaa_3d(Viewport::MSAA_4X);
```

Comment:
```suggestion
	viewport->set_msaa_3d(Viewport::MSAA_4X);
	// Use supersampling to further reduce aliasing.
	viewport->scaling_3d_scale(2.0);
```

This helps quite a bit with aliasing in my experience (I've tested it locally). The performance impact should be limited as the material preview is a low-resolution viewport compared to the rest of the 3D scene.